src/styles/components/card.scss

```diff
@@ -19,7 +19,7 @@ $block: ns(card);
     $tn-card-media-width: 75px !default;
     $tn-card-color-border: tn-color(neutral,2) !default;
     $tn-card-color-disabled: tn-color(text,3) !default;
-    $tn-card-background-color: tn-color(inverse) !default;
+    $tn-card-background-color: tn-color(text-inverse) !default;
```

Comment:
Various points here:

1. I'm fine adding a new group of colors named `background`. That can include White. All themeable colors should be in color group anyway.
2. `text` is fine as it is, when is only used for text. `text` <> `text-inverse` is pretty clear what the colors in those groups are used for.

We cannot change names all the time. We need to stick with the scalability of the color system we selected. The system scale in two directions: more shades in a group, more colors groups.